### PR TITLE
add Renebutcher cypress new file

### LINKED ISSUE #30: Create a file for ReneButcher Cypress
- [x] Create a branch
- [x] Add a commit
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by ReneButcher**
closes #30

### CODE REVIEW

**Comment by Rekha-Cnc** [comment]
Code review done, looks good.

File: renebutcher-cypress.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Cypress, CA
+
+### Places to Eat
```
Comment (Rekha-Cnc) [inline renebutcher-cypress.md:3]: good recommendations!

Comment (RLMorrison) [reply renebutcher-cypress.md:3]: Code review complete.

**Review by RLMorrison — APPROVED** [review]

File: renebutcher-cypress.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- City Tacos
```
Comment (Rekha-Cnc) [inline renebutcher-cypress.md:4]: :+1:

Comment (Rekha-Cnc) [reply renebutcher-cypress.md:4]: good one

**Review by Rekha-Cnc — APPROVED** [review]
great places to eat :taco:

**Pull request merged**